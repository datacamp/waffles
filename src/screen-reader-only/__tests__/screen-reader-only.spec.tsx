import { render } from '@testing-library/react';

import { ScreenReaderOnly } from '../index';

describe('ScreenReaderOnly', () => {
  it('renders snapshot', () => {
    const { container } = render(
      <p>
        Some regular visible text{' '}
        <ScreenReaderOnly>
          and visually hidden part visible only for screen readers
        </ScreenReaderOnly>
      </p>,
    );

    const content = container.firstChild;

    expect(content).toMatchSnapshot();
  });
});
