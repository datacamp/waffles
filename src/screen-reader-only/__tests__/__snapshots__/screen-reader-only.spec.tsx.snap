// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenReaderOnly renders snapshot 1`] = `
.emotion-0 {
  position: absolute;
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  width: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  white-space: nowrap;
  word-wrap: normal;
}

<p>
  Some regular visible text
   
  <span
    class="emotion-0"
  >
    and visually hidden part visible only for screen readers
  </span>
</p>
`;
